fvm/executionParameters.go

```diff
@@ -11,6 +11,91 @@ import (
 	"github.com/onflow/flow-go/fvm/utils"
 )
 
+func setMeterParameters(env *commonEnv) error {
+	// Don't enforce limits while setting up limits.
+	env.sth.DisableAllLimitEnforcements()
```

Comment:
NOTE: This behaviour is new.  Previously, we were metering while setting up the meter ...